spec/requests/institution_visibility_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+RSpec.describe 'Insitution visiblity work access', type: :request, clean: true, multitenant: true do
+  let(:account) { create(:account) }
+  let(:account2) { create(:account) }
+  let(:tenant_user) { create(:user) }
+  let(:tenant2_user) { create(:user) }
+  let(:work) { create(:work, visibility: 'authenticated') }
+
+  before do
+    WebMock.disable!
+    Apartment::Tenant.create(account.tenant)
+    Apartment::Tenant.switch(account.tenant) do
+      Site.update(account: account)
+      tenant_user
+      work
+    end
+    Apartment::Tenant.create(account2.tenant)
+    Apartment::Tenant.switch(account2.tenant) do
+      Site.update(account: account2)
+      tenant2_user
+    end
+  end
+
+  after do
+    WebMock.enable!
+    Apartment::Tenant.drop(account.tenant)
+  end
+
+  describe 'as an end-user' do
+    it 'allows access for users of the tenant' do
+      login_as tenant_user, scope: :user
+      get "http://#{account.cname}/concern/generic_works/#{work.id}"
+      expect(response.status).to eq(200)
```

Comment:
I do not think this really matches what we should expect. I think the user should have to have a role. however, I think it makes sense to go ahead and expand this spec to cover signing the users up in each tenant instead of trying the declare trick on lines 21 and 15.  It will still fail "as is" but that is a spec that I feel like should pass.

Reply:
That makes sense.  I'll switch to user registration through requests instead of test setup.